engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -207,7 +209,9 @@ class WorkflowActor(workflowId: WorkflowId,
 
   when(WorkflowAbortingState) {
     case Event(x: EngineLifecycleStateCompleteResponse, data @ WorkflowActorData(_, Some(workflowDescriptor), _)) =>
-      finalizeWorkflow(data, workflowDescriptor, failures = None)
+      // TODO: PBE: some of the x-es have an actually execution & output stores.
+      // But do we want the finalization to operate on that data during state aborting?
```

Comment:
During this aborting state, we could potentially finalize based on already completed executions and outputs. On the next line though, we just finalize as if no execution nor outputs were generated.

Unlike the other classes extending `WorkflowExecutionActorResponse` (WEActorR) `WorkflowExecutionAbortedResponse` (WEAbortedR) is both a `EngineLifecycleStateCompleteResponse` (ELSCR) and a WEActorR. Any WEActorR ~~will~~ could contain an `ExecutionStore` and `OutputStore` up until the point the workflow was aborted. However, the ELSCR trait is the only one we `case` match on though, for now. I don't really understand the current separation between the different classes implementing WEActorR and ELSCR, so this note is a hint to future implementor that one could get the execution and output stores if they change the matching type of `x`.